### ISSUE TITLE
HUGGING FACE  - runwayml/stable-diffusion-v1-5 - ERROR 404

### ISSUE THREAD

**Issue by HectorPenades** [foivospar/Arc2Face#29]
Hello,

I noticed that the model huggingface.co/runwayml/stable-diffusion-v1-5 is no longer available. Could you please let me know if this affects the functionality of the program in any way?

Thank you!

**Comment by foivospar** [comment]
Yes, either [stable-diffusion-v1-5/stable-diffusion-v1-5](https://huggingface.co/stable-diffusion-v1-5/stable-diffusion-v1-5) or [ashllay/stable-diffusion-v1-5-archive](https://huggingface.co/ashllay/stable-diffusion-v1-5-archive) can be used in place of the deprecated repository if the models are not already cached.
